bin/testUpdated.js

```diff
@@ -20,23 +20,22 @@
 const {execSync} = require('child_process');
 
 let output;
-console.info('Checking for updated packages');
+console.info('Checking for changed packages...');
+
 try {
-  output = execSync(`npx lerna updated`);
+  output = execSync(`npx lerna changed --all --json`);
```

Comment:
:+1:

Reply:
Good call! I opened a ticket to get information about their API: https://github.com/lerna/lerna/issues/2013